bincopy.py

```diff
@@ -611,9 +611,9 @@ def as_binary(self, minimum_address=None, padding=b'\xff'):
 
         for address, data in self.segments.iter():
             address //= self.word_size_bytes
-            res += padding * (address - current_maximum_address)
+            res += padding * (address - current_maximum_address)*self.word_size_bytes
```

Comment:
Addressed your comments.
Could you please check the unitests... it seems that it's something with line endings .. 
Thanks